config/config.go

```diff
@@ -92,6 +92,10 @@ type Config struct {
 	// PodSpecLogStrategy enables the logging of podspec on controller log.
 	PodSpecLogStrategy PodSpecLogStrategy `json:"podSpecLogStrategy,omitempty"`
 
+	// PodGCGracePeriodSeconds specifies the duration in seconds before the pods in the GC queue get deleted.
+	// Value must be non-negative integer. Defaults to zero, which indicates delete immediately.
+	PodGCGracePeriodSeconds int64 `json:"podGCGracePeriodSeconds,omitempty"`
```

Comment:
uint64 allowed?

Reply:
Updated this to be `*int64` to be consistent with the type of `DeleteOptions.GracePeriodSeconds`.